sunpy/map/mapbase.py

```diff
@@ -154,32 +154,34 @@ def __repr__(self):
 Measurement:\t {meas:0.0f}
 Obs Date:\t {date}
 dt:\t\t {dt:f}
-Dimension:\t [{xdim:d}, {ydim:d}]
-[dx, dy] =\t [{dx:f}, {dy:f}]
+Dimension:\t {dim}
+scale:\t\t [{dx}, {dy}]
 
 """.format(dtype=self.__class__.__name__,
            obs=self.observatory, inst=self.instrument, det=self.detector,
            meas=self.measurement, date=self.date, dt=self.exposure_time,
-           xdim=self.data.shape[1], ydim=self.data.shape[0],
+           dim=self.shape,
```

Comment:
This change transposes the reported dimensions of the map, which seems unintended.